01_financial-well-being/housing-value.qmd

```diff
@@ -0,0 +1,904 @@
+---
+title: "Housing Value"
+author: " Manu Alcalá Kovalski"
+date: now
+format:
+  html:
+    embed-resources: true
+    toc: true
+    toc_float: true
+    code-fold: show
+    code-tools: true
+editor_options: 
+  chunk_output_type: console
+execute:
+  warning: false
+---
+
+```{=html}
+<style>
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
+</style>
+```
+<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato" />
+
+This program creates the **Housing Value** metric defined as the ratio between the share of a community’s housing wealth held by a racial or ethnic group to the share of households of the same group metric
+
+Initially written by Jung Hyun Choi and updated by Manu Alcalá Kovalski in 2023-2024.
+
+## Process
+
+-   [Housekeeping](#housekeeping)
+-   [Read Data](#read-data)
+-   [Clean Data](#clean-data)
+-   [Merge Crosswalk](#merge-crosswalk)
+-   [Create Housing Value Metric](#create-metric)
+-   [Evaluate Results](#evaluate-results)
+-   [Export](#export)
+
+## Housekeeping {#sec-housekeeping}
+
+```{r}
+#| label: setup
+
+options(scipen = 999)
+librarian::shelf(tidyverse, ipumsr, here, glue, labelled, skimr, urbnthemes, tidylog, assertr)
+set_urbn_defaults(style = "print")
+```
+
+### Read Data {#sec-read-data}
+
+We extract the 1 year ACS files from the IPUMS API using `ipumsr`. You will need to have an API key in order to do so. If you don't have one, you can register to the [IPUMS USA](https://uma.pop.umn.edu/usa/user/new?return_url=https%3A%2F%2Fusa.ipums.org%2Fusa-action%2Fmenu) data collection and create an API key.
+
```

Comment:
There are a lot more steps to this (e.g., apply to use the data to have permission to use your API key, etc)-- how much detail do we think is needed here?

Reply:
I linked to the ipumsr vignette on [obtaining IPUMS data](https://tech.popdata.org/ipumsr/articles/ipums.html) since that's what I used it anyway!

Reply:
Perfect!